processes/fulfill_url_manifest.py

```diff
@@ -0,0 +1,166 @@
+import json
+import os
+import copy
+import logging
+from botocore.exceptions import ClientError
+
+from .core import CoreProcess
+from datetime import datetime, timedelta, timezone
+from model import Link
+from logger import createLog
+
+logger = createLog(__name__)
+
+class Fulfill_Process(CoreProcess):
+
+    def __init__(self, *args):
+        super(Fulfill_Process, self).__init__(*args[:4])
+
+        self.fullImport = self.process == 'complete' 
+        self.startTimestamp = None
+
+        # Connect to database
+        self.generateEngine()
+        self.createSession()
+
+        # S3 Configuration
+        self.s3Bucket = os.environ['FILE_BUCKET']
+        self.host = os.environ['DRB_API_HOST']
+        self.prefix = 'manifests/UofM/'
+        self.createS3Client()
+
+    def runProcess(self):
+        if self.process == 'daily':
+            startTimeStamp = datetime.now(timezone.utc) - timedelta(days=1)
+            self.get_manifests(startTimeStamp)
```

Comment:
From reading this function from here, it just looks as though we are getting manifests.  But this function has two responsibilities: 

1. Get manifests
2. Update manifests

I like following the single responsibility principle per function.

Reply:
Ok I can refactor the code to make `update_manifest `not be called in the `get_manifests` method but instead the runProcess method.

Reply:
From my perspective the method is mainly getting the manifests to provide for the  calls the update_manifests method to update the manifests. I think it's better to keep it like this for now and I'll keep what you said in mind